__tests__/postService.test.js

```diff
@@ -33,21 +34,27 @@ beforeAll(() => {
         };
     });
     postDAO.getPost.mockImplementation(async (id) => {
-        for (let i = 0; i < mockDatabase.length; i++){
-            if (mockDatabase[i].itemID == id){
+        for (let i = 0; i < mockDatabase.length; i++) {
+            if (mockDatabase[i].itemID == id) {
                 return {
                     $metadata: {
                         httpStatusCode: 200
                     },
-                    Item: mockPost1
+                    Item: mockDatabase[i]
+                };
+            } else {
```

Comment:
I don't think we need this else statement since it would only check the first object (and none of the other objects) in the database. Then, it would return a status code of 400 if the object is not the one we're looking for.

Reply:
You're right, this should be outside of the for loop.